linkquoter/linkquoter.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, bot: Red) -> None:
             },
             "webhooks": {
                 "converter": bool,
-                "description": "Toggle deleting of messages for automatic quoting.\n\nIf automatic quoting is enabled, then [botname] will also delete messages that contain links in them.",
+                "description": "Toggle showing the original Author of the quote instead of the bot when quoting",
```

Comment:
I think it would be better to use something like "Toggle sending message with the name and avatar of the Author of the quote (with webhooks).".

Reply:
No problem